doc/man3/EVP_PKEY_new.pod

```diff
@@ -121,21 +122,6 @@ data. The B<EVP_PKEY> structure will be initialised without any private key
 information. Algorithm types that support raw public keys are
 B<EVP_PKEY_X25519>, B<EVP_PKEY_ED25519>, B<EVP_PKEY_X448> or B<EVP_PKEY_ED448>.
 
-EVP_PKEY_new_CMAC_key_ex() works in the same way as
```

Comment:
Could you please instead of completely removing the description also for `EVP_PKEY_new_CMAC_key` just revert back to the text from 1.1.1 branch?

Reply:
I restored the 1.1.1 text, and fixed a bug describing how to do the new way.